src/Abstracts/WebsiteEvent.php

```diff
@@ -23,8 +24,14 @@ abstract class WebsiteEvent extends AbstractEvent
      */
     public $website;
 
-    public function __construct(Website &$website)
+    /**
+     * @var Hostname
+     */
+    public $hostname;
+
+    public function __construct(Website &$website, Hostname $hostname = null)
```

Comment:
```Hostname typehit should be
    public function __construct(Website &$website, ?Hostname $hostname = null)
```

Reply:
Optional function arguments aren't using the question mark, but a default value `null`, check the php documentation for more information:

http://php.net/manual/en/migration71.new-features.php

![image](https://user-images.githubusercontent.com/504687/53725588-97761880-3e6c-11e9-8037-86ec07e4ee5b.png)